ext/agent/main.go

```diff
@@ -18,26 +19,33 @@ var logger = logrus.New()
 var jscheduler, _ = gocron.NewScheduler()
```

Comment:
`jscheduler` doesn't say much about this variable, maybe we should name it `jobScheduler`